<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="TS_Login" time="32" tests="0" failures="1" errors="0">
   <testsuite name="TS_Login" tests="0" failures="1" errors="0" time="32" timestamp="2018-01-24 17:11:17" hostname="aekachaiboonruang - 192.168.3.21" id="Test Suites/TS_Login">
      <properties>
         <property name="deviceName" value=""/>
         <property name="devicePlatform" value=""/>
         <property name="logFolder" value="/Users/aekachaiboonruang/Katalon Studio/mobile/Reports/TS_Login/20180124_171113"/>
         <property name="logFiles" value="/Users/aekachaiboonruang/Katalon Studio/mobile/Reports/TS_Login/20180124_171113/appium-proxy-server.log, /Users/aekachaiboonruang/Katalon Studio/mobile/Reports/TS_Login/20180124_171113/appium.log, /Users/aekachaiboonruang/Katalon Studio/mobile/Reports/TS_Login/20180124_171113/execution0.log"/>
         <property name="attachments" value=""/>
         <property name="hostName" value="aekachaiboonruang - 192.168.3.21"/>
         <property name="hostAddress" value="192.168.3.21"/>
         <property name="os" value="Mac OS X 64bit"/>
         <property name="katalonVersion" value="5.2.0.1"/>
      </properties>
      <testcase name="Test Cases/Login iOS" classname="Test Cases/Login iOS" status="FAILED">
         <failure type="FAILED" message="Unable to start app at: '/Users/aekachaiboonruang/Desktop/DEV-Arbotena.app' (Root cause: com.kms.katalon.core.appium.exception.AppiumStartException: Fail to start Appium server in 30 seconds)"/>
         <system-out><![CDATA[2018-01-24 17:11:17 - [TEST_CASE][FAILED] - Test Cases/Login iOS: Test Cases/Login iOS FAILED because (of) Unable to start app at: '/Users/aekachaiboonruang/Desktop/DEV-Arbotena.app' (Root cause: com.kms.katalon.core.appium.exception.AppiumStartException: Fail to start Appium server in 30 seconds)

2018-01-24 17:11:17 - [TEST_STEP][FAILED] - startApplication: Unable to start app at: '/Users/aekachaiboonruang/Desktop/DEV-Arbotena.app' (Root cause: com.kms.katalon.core.appium.exception.AppiumStartException: Fail to start Appium server in 30 seconds)

2018-01-24 17:11:49 - [MESSAGE][FAILED] - Unable to start app at: '/Users/aekachaiboonruang/Desktop/DEV-Arbotena.app' (Root cause: com.kms.katalon.core.appium.exception.AppiumStartException: Fail to start Appium server in 30 seconds)]]></system-out>
         <system-err><![CDATA[2018-01-24 17:11:17 - [TEST_CASE][FAILED] - Test Cases/Login iOS: Test Cases/Login iOS FAILED because (of) Unable to start app at: '/Users/aekachaiboonruang/Desktop/DEV-Arbotena.app' (Root cause: com.kms.katalon.core.appium.exception.AppiumStartException: Fail to start Appium server in 30 seconds)]]></system-err>
      </testcase>
      <system-out><![CDATA[2018-01-24 17:11:17 - [TEST_SUITE][FAILED] - TS_Login: Test Cases/Login iOS FAILED because (of) Unable to start app at: '/Users/aekachaiboonruang/Desktop/DEV-Arbotena.app' (Root cause: com.kms.katalon.core.appium.exception.AppiumStartException: Fail to start Appium server in 30 seconds)]]></system-out>
      <system-err><![CDATA[2018-01-24 17:11:17 - [TEST_SUITE][FAILED] - TS_Login: Test Cases/Login iOS FAILED because (of) Unable to start app at: '/Users/aekachaiboonruang/Desktop/DEV-Arbotena.app' (Root cause: com.kms.katalon.core.appium.exception.AppiumStartException: Fail to start Appium server in 30 seconds)]]></system-err>
   </testsuite>
</testsuites>
